### PR TITLE
Aihub examples

### PR DESCRIPTION
**Pull request by post2web**
This PR contains the other 12 notebooks and a README.md. I also made small fixes to the already merged notebook.

### CODE REVIEW

File: notebooks/samples/aihub/knn_classification/knn_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,16305 @@
+{
```
Comment (gogasca) [inline notebooks/samples/aihub/knn_classification/knn_classification.ipynb]: <p>Remove tf.contrib references</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Comment by gogasca** [comment]
@post2web Thanks for the contributions, all seems to be good, just a common item across notebooks:
1) Remove tf.contrib references
2) Create models using TF 1.15

File: notebooks/samples/aihub/knn_classification/knn_classification.ipynb (hunk 1 above)
Comment (post2web) [reply notebooks/samples/aihub/knn_classification/knn_classification.ipynb]: <p>The <code>tf.contrib</code>  reference is in an HTML file that was generated after training. That HTML file is embedded into the notebook and shown as cell output. In order for me to remove that reference, we need to modify all of the ML components, do another release and rerun the training of all of the notebooks. </p><p> </p><p>This project may end at the end of the week and I am not sure if we will have the time to do that. Do you think we can merge them with the <code>tf.contrib</code>  for now?</p>

Comment (gogasca) [reply notebooks/samples/aihub/knn_classification/knn_classification.ipynb]: Ok, that's fine.

**Comment by gogasca** [comment]
@post2web there is some conflicts that is preventing merge

**Comment by post2web** [comment]
@gogasca this PR should be ready to merge.

**Pull request merged**